apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -737,28 +740,43 @@ where
                             self.storage.last_height,
                         ) {
                             response.log = String::from(VALID_MSG);
+                            // validator set update votes should be decided
+                            // as soon as possible
+                            response.priority = i64::MAX;
                         } else {
                             response.code = 1;
-                            response.log = String::from(
-                                "Invalid validator set update vote extension",
+                            response.log = format!(
+                                "{INVALID_MSG}: Invalid validator set update \
+                                 vote extension",
                             );
                         }
                     }
                     Ok(TxType::Protocol(ProtocolTx { tx, .. })) => {
                         response.code = 1;
                         response.log = format!(
-                            "The following protocol tx cannot be added to the \
-                             mempool: {tx:?}"
+                            "{INVALID_MSG}: The following protocol tx cannot \
+                             be added to the mempool: {tx:?}"
                         );
                     }
-                    // `process_tx` errors are handled by
-                    // `Shell::finalize_block`
-                    _ => response.log = String::from(VALID_MSG),
+                    Ok(TxType::Wrapper(_) | TxType::Raw(_)) => {
+                        response.log = String::from(VALID_MSG);
+                    }
```

Comment:
Should we be allowing `TxType::Raw` into the mempool? I thought all user txs would have to be of `TxType::Wrapper`

Reply:
we were allowing raw transactions to be included before, but yeah, they're not even affecting state in FinalizeBlock

Reply:
No, we don't allow TxType::Raw

Reply:
Or, I should say, they will be rejected by process proposal. So we shouldn't include them